supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc

```diff
@@ -31,6 +31,17 @@
 
 *See also*:
 
+[discrete]
+[[weak-dependencies]]
+==== image:images/yes.png[yes] Weak dependencies (noun)
```

Comment:
I'm assuming that this is correct and how you use them in RHEL docs, but I at least want to say that it seems like several of these terms that are capitalized maybe don't need to be. I understand that some things do need to be capitalized for one reason or another, but this seems like a generic enough term that maybe it should be lowercased. Just something to consider if it hasn't truly been discussed whether these all need to be capitalized.

Again, fine as is if that's how you use them. But still just want to point out for the sake of avoiding capitalizing too many things that don't need to be.